### PR TITLE
Added Form_Custom demo page component

### PR DESCRIPTION
**Pull request by protoEvangelion**

### CODE REVIEW

**Comment by alexandervalencia** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/pages/docs/components/form_custom.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+		<input id="radioStacked4" name="radio-stacked" type="radio" class="custom-control-input">
+		<span class="custom-control-indicator"></span>
+		<span class="custom-control-description">Or toggle this other custom radio</span>
+	</label>
+</div>
+```
+
+</article>
+
+<script>
```
Comment (alexandervalencia) [inline src/pages/docs/components/form_custom.md]: Script in markdown files should start flush to the wall with no line breaks.
i.e.
```xml
<script>
{literal}
$('.clay-site-custom-checkbox-indeterminate').prop('indeterminate', true);
$('[data-clay-site-toggle="file"]').on('change', function(event) {
	var path = $(this).val();
	var name = path.substring(path.lastIndexOf("\\") + 1, path.length);
	var input = $(this).data('clay-site-content');
	$(input).val(name);
});
{/literal}
</script>
```

File: src/pages/docs/components/form_custom.md [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+{literal}
+	$('.clay-site-custom-checkbox-indeterminate').prop('indeterminate', true);
```
Comment (alexandervalencia) [inline src/pages/docs/components/form_custom.md]: These function are not being called correctly. You'll need to find another way to accomplish this.

**Comment by alexandervalencia** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/pages/docs/components/form_custom.md (hunk 2 above)
Comment (protoEvangelion) [inline src/pages/docs/components/form_custom.md]: What do you mean the functions are not being called correctly?

Comment (protoEvangelion) [inline src/pages/docs/components/form_custom.md]: Would using vanilla JS to accomplish the same thing be acceptable?

Comment (alexandervalencia) [inline src/pages/docs/components/form_custom.md]: I mean, the code is not working as written/intended. You're running into the same issue I was earlier, the jquery is targeting an element that gets overwritten by the incremental DOM. You'll need to find another way to make it work.

Comment (protoEvangelion) [inline src/pages/docs/components/form_custom.md]: I see what you are saying I will go ahead and fix that ;)

**Pull request closed without merging**